fuel/src/main/kotlin/com/github/kittinunf/fuel/toolbox/HttpClient.kt

```diff
@@ -27,10 +27,12 @@ import kotlin.coroutines.resume
 import kotlin.coroutines.resumeWithException
 import kotlin.coroutines.suspendCoroutine
 
-internal class HttpClient(
+class HttpClient(
     private val proxy: Proxy? = null,
+    private var stethoHook: StethoHook? = null,
```

Comment:
It's really not private, but this disallows setting/resetting it later. Make public instead.